homework13.js

```diff
@@ -0,0 +1,82 @@
+// # Class Exercises
+//
+// 1.
+//   * Create a Human class that takes in a name and age.
+//   * Add the function `ageOneYear` that ages the human.
+//   * Add the function `eating`, that logs "mmm, mmm, mmm, I'm love'n it".
+//   * Create an instance of the Human class.
+//     * console log your humans age
+//     * call ageOneYear on your human
+//     * console log their age again.
+//     * call eating on your human.
+
+class Human {
+  constructor (name, age) {
+    this.name = name;
+    this.age = age;
+  }
+  ageOneYear() {
+    this.age++
+    return this.age
+  }
+  eating() {
+    return "mmm, mmm, mmm, i'm loving it!"
+  }
+}
+let jean = new Human("Jean", 27)
+jean.age;
```

Comment:
Nitpick but when a question calls for you to log something, thatt means to console.log it within the function itself. In the case of your eating function, it should console.log instead of returning.